astlib/parser.mli

```diff
@@ -0,0 +1,24 @@
+(* The type of tokens. *)
```

Comment:
This comment should give an overview of the module.

Reply:
Good catch! For now, I've simply removed that comment. I've had a look at the ppx-universe and we don't even need this module! I'll open a separate PR removing it. I will also write and open another separate PR adding a bit more of documention to Astlib in general.